app/src/main/res/values/strings.xml

```diff
@@ -148,8 +148,8 @@
     <string name="restore_title">Restore from backup</string>
     <string name="restore_choose_restore_set">Choose a backup to restore</string>
     <string name="restore_restore_set_times">Last backup %1$s · First %2$s.</string>
-    <string name="restore_back">Don\'t restore</string>
-    <string name="restore_skip">Skip restoring apps</string>
+    <string name="restore_skip">Don\'t restore</string>
+    <string name="restore_skip_apps">Skip restoring apps</string>
```

Comment:
This is likely to mess with translations. Maybe don't rename or at least chose completely new names?

Reply:
I agree.